scalameta/io/js/src/main/scala/scala/meta/internal/io/JSIO.scala

```diff
@@ -5,25 +5,38 @@ import scala.scalajs.js
 import scala.scalajs.js.annotation.JSImport
 import scala.scalajs.js.annotation.JSImport.Namespace
 
-/** Facade for npm package "shelljs".
+/** Facade for the native nodejs process API
   *
-  * @see https://www.npmjs.com/package/shelljs
+  * The process object is a global that provides information about, and
+  * control over, the current Node.js process. As a global, it is always
+  * available to Node.js applications without using require().
+  *
+  * @see https://nodejs.org/api/process.html
   */
 @js.native
-@JSImport("shelljs", Namespace)
-object JSShell extends js.Any {
```

Comment:
Very nice to get rid of this, I was a total nodejs noob when I started writing these facades.